src/modules/gridUtil.js

```diff
@@ -14,18 +14,16 @@ import {
 import { isSmallScreen } from './smallScreen.js'
 
 export const GRID_COMPACT_TYPE = 'vertical' // vertical | horizonal | null
-export const GRID_ROW_HEIGHT_PX = 10
-export const MARGIN_PX = [10, 10]
+export const GRID_ROW_HEIGHT_PX = 16
+export const MARGIN_PX = [4, 4]
 
 const SM_SCREEN_MIN_ITEM_GRID_HEIGHT = 13 // minimum of ~320px
 export const SM_SCREEN_GRID_COLUMNS = 1
-export const MARGIN_SM_PX = [0, 16]
+export const MARGIN_SM_PX = [4, 4]
 export const GRID_PADDING_PX = [0, 0]
-// sum of left+right padding of dashboard-wrapper (App.css)
-export const DASHBOARD_WRAPPER_LR_MARGIN_PX = 32
```

Comment:
Note that this constant could be removed because we now work with that `div.contentWrap` which already has some whitespace around it and will shrink when a scrollbar appears.